### DISCUSSION TITLE
Check if translation is up-to-date

### DISCUSSION THREAD

**Discussion by foxtrotcz** [craftcms/cms#13732] [Ideas]
Hi,
lets say you have site with 2 language versions. You have content in original language and translated content.
If you make changes in the original content you have to remember to also update translation.
It could improve ease of use if Craft tracked if the translation needs updating or if it's already updated.

Just some small icon next to entry or next to updated field would be fine. Icon that would mean "You edited this content but you didnt edit translation. Maybe you need to update your translation."

Thanks.

**Comment by piotrpog** [comment]
I think there could be something like option to mark specific language versions of specific entry for edition. By enabling this to the specific entry, we could then edit specific language version and when saving check some checkbox like "language version updated" (just saving would not by default mean that we are finished with this language version).

We could also se in the sidebar list of language versions and their status, "updating completed" or not. 

This could help coordinate efforts between translators, or even when you have some word document with multiple versions of texts and you just paste them into 10 language versions of the entry in the row.
